c/src/point_one/polaris/polaris.c

```diff
@@ -236,11 +282,18 @@ void Polaris_Free(PolarisContext_t* context) { CloseSocket(context, 1); }
 
 /******************************************************************************/
 void Polaris_SetLogLevel(int log_level) {
-#if !defined(POLARIS_NO_PRINT)
+#if !POLARIS_NO_PRINT
```

Comment:
are you sure we dont want ifndef....this just invest the value of POLARIS_NO_PRINT (which i guess by default is 0?)

Reply:
This is a little more flexible in practice. `!defined(POLARIS_NO_PRINT)` would disable prints if the `POLARIS_NO_PRINT` macro is defined, no matter what its value is.

If you don't use `defined()` and just use the macro in an expression like `!POLARIS_NO_PRINT` and the macro is not defined, the compiler will treat it as 0. So in this case you get:
- Don't define `POLARIS_NO_PRINT` at all -- Prints enabled
- `#define POLARIS_NO_PRINT 0` -- Prints enabled
- `#define POLARIS_NO_PRINT 1` -- Prints disabled

This makes it a little easier to turn prints on/off in a `CMakeLists.txt` file for example by having it change the value it uses in a `-DPOLARIS_NO_PRINT=${foo}` statement instead of having to remove the line entirely.